src/main/java/org/chtracker/web/AbortiveTreatmentTypeController.java

```diff
@@ -0,0 +1,28 @@
+package org.chtracker.web;
+
+import org.chtracker.dao.metadata.AbortiveTreatmentType;
+import org.chtracker.dao.metadata.AbortiveTreatmentTypeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/abortive-treatment-types")
+public class AbortiveTreatmentTypeController {
+
+	private final AbortiveTreatmentTypeRepository abortiveTreatmentTypeRepository;
+
+	@Autowired
```

Comment:
You are already dependency injecting with constructor @Autowired annotation seems pointless in here. 
It compiles and runs without it.

Reply:
that's annotation on constructor, without it the field won't be injected.